Demos/Eyes/EyesWidget.cpp

```diff
@@ -105,17 +106,11 @@ Gfx::Point EyesWidget::pupil_center(Gfx::Rect& eyeball_bounds) const
     if (dx != 0 && abs(dx) >= abs(dy)) {
         double slope = dy / dx;
         double slope_squared = slope * slope;
-        max_distance_along_this_direction = 0.25 * sqrt(
-            (slope_squared + 1) /
-            (1 / width_squared + slope_squared / height_squared)
-        );
+        max_distance_along_this_direction = 0.25 * sqrt((slope_squared + 1) / (1 / width_squared + slope_squared / height_squared));
     } else if (dy != 0 && abs(dy) >= abs(dx)) {
         double slope = dx / dy;
         double slope_squared = slope * slope;
-        max_distance_along_this_direction = 0.25 * sqrt(
-            (slope_squared + 1) /
-            (slope_squared / width_squared + 1 / height_squared)
-        );
+        max_distance_along_this_direction = 0.25 * sqrt((slope_squared + 1) / (slope_squared / width_squared + 1 / height_squared));
```

Comment:
why.
@bugaevc why.

Reply:
Because this is way longer than 80 characters, not quite readable, and clang-format apparently doesn't want to care. I'd leave it as is.

Reply:
I'm not touching this.
Not my problem, run clang-format at your own risk.

Reply:
at least `// clang-format {off,on}` so people don't have to mess with `git add -p`...